packages/api/cms-api/src/warnings/decorators/create-warnings.decorator.ts

```diff
@@ -0,0 +1,38 @@
+import { type AnyEntity } from "@mikro-orm/postgresql";
+import { type CustomDecorator, SetMetadata, type Type } from "@nestjs/common";
+
+import { type CreateWarningInput } from "../dto/create-warning.input";
+
+interface BulkCreateWarningInput {
+    warnings: CreateWarningInput[];
+    tableRowId: string;
```

Comment:
`targetId` be more matching with the existing apis. Or targetEntityId? or entityId?

Reply:
I changed it to targetId, or do you prefer another naming?
8c309cfd81935ae70337ed8bbe2d7d59d3e62ef9

Reply:
targetId is fine by me.